### ISSUE TITLE
Git Status on Grid

### ISSUE THREAD

**Issue by typkrft** [ogham/exa#927]
This program is great. I would humbly as you to consider adding git status symbols to the grid view. Using small screens makes working with large directories output hard to parse at a glance. 

Also a minor nitpick some of the icons are a little close to the names a little space would be nice. Maybe this is just my setup?

Both of these things are subjective and may no warrant the effort. Either way thank you. 
<img width="815" alt="Screen Shot 2021-08-08 at 9 41 43 PM" src="https://user-images.githubusercontent.com/12100960/128652844-3a0e42d4-5eac-4372-8ca1-aef274e44372.png">

**Comment by ariasuni** [comment]
For your icon issue, extract from `man exa`:

> ENVIRONMENT VARIABLES
> […]
> `EXA_ICON_SPACING`
>  Specifies the number of spaces to print between an icon (see the `--icons' option) and its file name.
>
> Different terminals display icons differently, as they usually take up more than one character width on screen, so there’s no “standard” number of spaces that exa can use to separate an icon from text. One space may place the icon too close to the text, and two spaces may place it too far away. So the choice is left up to the user to configure depending on their terminal emulator.

**Comment by typkrft** [comment]
Makes complete sense. Thank you very much for that piece.

**Comment by zachvalenta** [comment]
Another vote for ability to add Git status to metadata table in grid view.

**Comment by ariasuni** [comment]
There’s a reaction feature (smiley face icon, next to the three-dot icon) that you can use, so it avoids adding yet another notification to my endless list of them.

I guess any feature could be implemented for the grid view mode, but I don’t personally have the time to do it.
